crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -89,6 +89,29 @@ impl<I: PhaseItem> RenderPhase<I> {
             draw_function.draw(world, render_pass, view, item);
         }
     }
+
+    /// Renders a range of the [`PhaseItem`]s using their corresponding draw functions.
```

Comment:
```suggestion
    /// Renders all  [`PhaseItem`]s in the provided `range` (based on their index in `self.items`) using their corresponding draw functions.
```

"Range" is pretty overloaded as a term, and it's very helpful to rephrase this sort of thing when writing docs.